spec/support/mongoid.rb

```diff
@@ -1,3 +1,39 @@
+module Mongoid
+  module Matchers
+    class HaveFieldMatcher
```

Comment:
i'm guessing, not entirely sure. I poked around the git history of that project a little but couldn't find anything. do you have a preference for another way of doing it?